src/cmd/root.go

```diff
@@ -359,6 +364,31 @@ func newSubIDFileError() error {
 	return errors.New(errMsg)
 }
 
+func setUpConfiguration() (bool, error) {
+	logrus.Debug("Setting up configuration")
+
+	configFiles := []string{
+		"/etc/containers/toolbox.conf",
+		fmt.Sprintf("%s/.config/containers/toolbox.conf", currentUser.HomeDir),
+	}
+
+	viper.SetConfigType("toml")
+
+	for _, configFile := range configFiles {
+		viper.SetConfigFile(configFile)
+
+		if err := viper.MergeInConfig(); err != nil {
+			if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+				logrus.Debugf("Setting up configuration: file %s not found", configFile)
+			} else {
+				logrus.Debugf("Setting up configuration: failed to merge file %s: %s", configFile, err)
+			}
+		}
+	}
+
+	return true, nil
```

Comment:
There's no need to return `error` since we handle errors gracefully and do not fail early in any case.

Reply:
Yes, you are right. I will drop it.